public/app/plugins/panel/alertmanager-notifications/module.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { PanelPlugin } from '@grafana/data';
+import { AMNotificationsOptions } from './types';
+import { AMNotifications } from './AMNotifications';
+import { AlertManagerPicker } from 'app/features/alerting/unified/components/AlertManagerPicker';
+
+export const plugin = new PanelPlugin<AMNotificationsOptions>(AMNotifications).setPanelOptions((builder) => {
+  return builder
+    .addTextInput({
+      name: 'Labels',
+      path: 'labels',
+      category: ['Filter'],
+    })
+    .addCustomEditor({
+      name: 'Alertmanager',
+      path: 'alertmanager',
+      id: 'alertmanager',
+      defaultValue: null,
```

Comment:
```suggestion
      defaultValue: GRAFANA_RULES_SOURCE_NAME,,
```

Set Grafana as default rather than empty